packages/callout/src/Callout.tsx

```diff
@@ -21,33 +19,33 @@ export const Variant = {
 export type Variant = typeof Variant[keyof typeof Variant];
 
 const baseStyle = css`
-  background-color: ${uiColors.white};
-  border-radius: 6px;
-  box-shadow: 0px 2px 5px 0 ${transparentize(0.9, uiColors.gray.dark3)};
-  overflow: hidden;
+  font-family: Euclid Circular A, ‘Helvetica Neue’, Helvetica, Arial, sans-serif; // TODO: Refresh – remove when fonts are updated
+  background-color: ${palette.white};
+  border-radius: 12px;
   position: relative;
 
   &:after {
     content: '';
     position: absolute;
-    width: 4px;
-    left: 0px;
+    width: 13px;
+    left: -1px;
     top: -1px;
     bottom: -1px;
-    border-radius: 6px 0px 0px 6px;
+    border-radius: 12px 0px 0px 12px;
   }
 `;
 
 const headerStyle = css`
-  font-family: Akzidenz, ‘Helvetica Neue’, Helvetica, Arial, sans-serif;
   font-size: 12px;
-  font-weight: bold;
-  letter-spacing: 0.8px;
+  font-weight: 700;
+  letter-spacing: 1px;
   line-height: 16px;
-  padding: 12px 24px 12px 52px;
+  padding: 12px 24px 12px 54px;
   position: relative;
   text-transform: uppercase;
   width: 100%;
+  border-top-left-radius: 12px;
+  border-top-right-radius: 11px;
```

Comment:
You should be able to achieve this with `overflow: hidden` on the parent

Reply:
I avoided `overflow: hidden` on the parent since the right border should not be shown. The colored bar should be fully flush with the edge of the container. It doesn't look like that in figma but Alven and I spoke about it in slack.
<img width="164" alt="Screen Shot 2022-01-24 at 10 47 30 AM" src="https://user-images.githubusercontent.com/5740159/150815915-675b4c06-84c6-454f-97ad-d0c489b278d2.png">